Fix send page layout [MAILPOET-5734]

NeosinneR opened the pull request:
## Description

_N/A_

## Code review notes

I decided not to use "mailpoet-main-container" because it's too wide, for two 368px columns.

## QA notes

_N/A_

## Linked PRs

_N/A_

## Linked tickets

[MAILPOET-5734]

## After-merge notes

_N/A_

## Tasks

- [x] 🚫 I followed [best practices](https://codex.wordpress.org/I18n_for_WordPress_Developers) for translations
- [x] 🚫 I added sufficient test coverage
- [x] 🚫 I embraced TypeScript by either creating new files in TypeScript or converting existing JavaScript files when making changes


[MAILPOET-5734]: https://mailpoet.atlassian.net/browse/MAILPOET-5734?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

JanJakes commented on mailpoet/assets/css/src/components-plugin/_forms.scss:
```diff
@@ -75,8 +75,13 @@ progress::-moz-progress-bar {
+    grid-template-columns: $grid-column $grid-column;
+    max-width: 976px;
+  }
+
```
> I decided not to use "mailpoet-main-container" because it's too wide, for two 368px columns.

:+1:, but do you think this is a good place to fix it (plus the `!important`)? I would argue that a form from "components" doesn't need to know about a specific email form. What I mean is that rather than `.mailpoet-form-grid { .mailpoet-form-send-email & { ... } }` here we could do `.mailpoet-form-send-email { .mailpoet-form-grid { ... } }` in another place.

Or maybe even easier, add a new `.mailpoet-form-send-email-container`...

NeosinneR replied:
That's fair. I've done it as you suggested and also moved out similarly component-specific style and `!important` with it.

NeosinneR committed:
Fix send page layout

I decided not to use "mailpoet-main-container" because it's too wide, for two 368px columns.

[MAILPOET-5734]

NeosinneR committed:
Move "send form"-specific styles out of generic form styles

[MAILPOET-5734]

JanJakes reviewed (approved):
Thanks!

Aschepikov reviewed (approved):
GJ!
